.gitignore

```diff
@@ -4,3 +4,20 @@ prboom2/data/prboom.wad
 prboom2/release
 /build*
 /dependencies*
+CMakeCache.txt
+CMakeFiles
+Makefile
+arch.c
+cmake_install.cmake
+config.h
+data/CMakeFiles
+data/Makefile
+data/cmake_install.cmake
+prboom2/data/rdatawad
+glboom-plus
+prboom-plus
+prboom-plus-game-server
+prboom-plus.wad
+src/CMakeFiles
+src/Makefile
+src/cmake_install.cmake
```

Comment:
There is no need to add all this. Please use designated build directories named `build*` instead. While doing in-tree build isn't prohibited, it's not recommended either.

Reply:
Should the documentation not say something about this? The INSTALL file still talks about using GNU autoconf.

Reply:
If you can update it, please do ;)